schemas/hotelSchema.js

```diff
@@ -0,0 +1,61 @@
+const Joi = require('joi')
+const mongoose = require('mongoose')
+import { Room } from './roomSchema'
+import { Address } from './addressSchema'
+import { Rate } from './rateSchema'
+
+const hotelSchema = new mongoose.Schema({
+  hotelId: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
+  ownerId: {
+    type: mongoose.Types.ObjectId,
+    required: true,
+  },
+  localization: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'Addres',
+    },
+  ],
+  phoneNumber: {
+    type: Number,
+    required: true,
+  },
+  name: {
+    type: String,
+    required: true,
+  },
+  clientsRates: [{ type: mongoose.Schema.Types.ObjectId, ref: 'Rate' }],
+  email: {
+    type: String,
+    required: true,
+  },
+  description: {
+    type: String,
+    required: true,
+  },
+  rooms: [{ type: Schema.Types.ObjectId, ref: 'Room' }],
+})
+
+const Hotel = mongoose.model('Hotel', hotelSchema)
+
+const validateHotel = (hotel) => {
+  const schema = Joi.object({
+    hotelId: Joi.ObjectId().required(),
+    ownerId: Joi.ObjectId().required(),
+    localization: Joi.object(Address),
+    phoneNumber: Joi.number,
```

Comment:
Should have min length

Reply:
I don't think there's such thing as length for number, I can set it to e.g. .min(111111111)